tests/lax_control_flow_test.py

```diff
@@ -28,14 +28,14 @@
 
 import jax
 from jax._src import core
+from jax._src import maps
```

Comment:
Can this be from jax._src.maps import xmap?

Reply:
Done